schemas/2018-02-01/Microsoft.Web.json

```diff
@@ -0,0 +1,7804 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2018-02-01/Microsoft.Web.json#",
```

Comment:
Please attach the console output when running `npm run generate-single ...` for us to review